cypress/pages/index.jsx

```diff
@@ -0,0 +1,64 @@
+import selectors from "../selectors/Login";
+
+class LoginPage {
```

Comment:
Let's remove the old index.jsx LoginPage now that you verified the new LoginPage one works